modules/ui/src/main/scala/lucuma/ui/forms/FormInputEV.scala

```diff
@@ -83,12 +89,15 @@ object FormInputEV {
 
   @Lenses
   final case class State(
-    curValue:  String,
-    prevValue: String,
-    errors:    Option[NonEmptyChain[NonEmptyString]]
+    displayValue: String,
+    modelValue:   String,
+    cursor:       Option[(Int, Int)],
+    lastKeyCode:  Int,
```

Comment:
Do we want to avoid rerender when `cursor` or `lastKeyCode` change?

Reply:
If we want to avoid said rerender, we can either: store in `private var`s inside backend; or define a `shouldComponentUpdate` in the component, or use `Reusability` (might be tricky to define it for `Props` and may not be worth the trouble since `FormInput` (probably) has some logic for that).

Reply:
I will try something like this and submit in a separate PR so my approach can more easily be discussed.